tests/native_tests/language.hy

```diff
@@ -1019,12 +1019,18 @@
 
 (defn test-xor []
   "NATIVE: test the xor macro"
-  (let [xor-both-true (xor True True)
-        xor-both-false (xor False False)
-        xor-true-false (xor True False)]
-    (assert (= xor-both-true False))
-    (assert (= xor-both-false False))
-    (assert (= xor-true-false True))))
+
+  ; Test each cell of the truth table.
+  (assert (is (xor False  False) False))
+  (assert (is (xor False True)  True))
+  (assert (is (xor True  False) True))
+  (assert (is (xor True  True)  False))
+
+  ; Same thing, but with numbers.
+  (assert (is (xor 0 0) 0))
```

Comment:
might be clearer as `(assert (is (xor False 0) 0))`, proving the second argument is returned, rather than the first.